libs/file/Makefile

```diff
@@ -55,7 +55,9 @@ CONFIGURE_ARGS += \
 	--enable-bzlib \
 	--enable-xzlib \
 	--enable-zlib \
+	--enable-zstdlib \
```

Comment:
What is the purpose to have it enabled by default? What is the size difference?
Also, on the other hand, we can have it disabled by default, so no need to add additional dependency.

Reply:
Facebook's Zstdlib was added in version 5.44 as an additional compression/decompression option and since zstdlib is already part of OpenWRT it would be fine if file and libmagic were built with support for all possible compressors/decompressions.
But yes, the compiled libzstd.so.1.5.2 file on the disk for the arm_cortex-a7+neon-vfpv4_musl_eabi target is a little over 700kB, which makes me wonder if it's really a good idea to activate zstdlib support.

I can deactivate it again - shall I?

Reply:
> I can deactivate it again - shall I? 

just deactivate it, (unless there is a really good reason to add the new functionality requiring an additional 700 kB lib)

Reply:
I agree - and i have deactivated zstdlib.